packages/release-trigger/src/bot.ts

```diff
@@ -64,7 +64,8 @@ async function doTriggerWithLock(
   pullRequest: PullRequest,
   token: string,
   logger: GCFLogger,
-  installationId: number
+  installationId: number,
+  multiScmName?: string
```

Comment:
Add @params to the function's doc comment for installationId and multiScmName.

Reply:
Done.